.drone/travis_script.sh

```diff
@@ -7,7 +7,17 @@ source ~/envs/cesium/bin/activate
 
 section "Tests"
 
-make ${TEST_TARGET}
+if [[$COVERAGE == 1]]
+
+then
+    echo "Python version: 3.4/3.5"
+    echo "Running coverage.py"
+    nosetests -v --exe --with-coverage
+else
+    echo "Python version: 2.x"
+    echo "Coverage not set up for python 2.x"
+    make ${TEST_TARGET}
```

Comment:
It looks like TEST_TARGET is unused.  Let's remove it from .travis.yml, and then do something like this:

```
if [[ $COVERAGE == 1 ]]; then
    NOSE_FLAGS="--with-coverage"
fi

nose -v $NOSE_FLAGS cesium
```